Kernel/System/ImportExport/FormatBackend/JSON.pm

```diff
@@ -280,6 +280,7 @@ sub ExportDataSave {
     return $JSONObject->Encode(
         Data   => $Param{ExportDataRow},
         Pretty => $Pretty,
+        Sort   => 1,
```

Comment:
The parameter should be 'SortKeys' instead of 'Sort'

Reply:
Fixed, thank you =)